content/docs/command-reference/checkout.md

```diff
@@ -69,6 +68,10 @@ be pulled from remote storage using `dvc pull`.
 
 ## Options
 
+- `--summary` - in addition to checking out DVC-tracked data, display a short
```

Comment:
I think it is sill confusing. This option just changes the format of the output (but even w/o it checkout outputs stats).

Reply:
There are no stats without this flag last time I checked. I tried it before writing this. Let me double check...

Reply:
Hm I was wrong. In fact --summary has a shorter output than regularly. Updating... ⏳

Reply:
Done in 5d29f1d7